controllers/topics/index.js

```diff
@@ -194,6 +201,7 @@ const followTopicV2 = async (req, res) => {
       userTopicService.getFollowTopicStatus(secondDetailUser.user_id, topic_id)
     ]);
 
+    let topicInvitationIds = [];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

An array `topicInvitationIds` has been initialized to store the IDs of topic invitations. This will be used later in the code to handle topic invitation channels.
```diff
+    let topicInvitationIds = [];
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->